experiments.json

```diff
@@ -1,7 +1,7 @@
 {
   "bookmark-history-menu": {
     "match": {
-      "appId": "^org.mozilla.fennec$"
+      "appId": "^org.mozilla.fennec*|^org.mozilla.firefox_beta$"
```

Comment:
The `org.mozilla.fennec*` line matches Nightly (`org.mozilla.fennec`), Aurora (`org.mozilla.fennec_aurora`), and all local builds.